source/plugin/plugin-meta.rst

```diff
@@ -27,7 +29,16 @@ You can define the additional (optional) plugin metadata on your ``@Plugin`` ann
         description = "This is a very cool plugin I made for me",
         url = "http://example.com",
         authors = {"Spongie", "FLARD"},
-        dependencies = @Dependency(id = "otherplugin", optional = true))
+        dependencies = {@Dependency(id = "otherplugin", optional = true)})
+
+For every ``@Dependency`` you provide, you may also give a ``version`` attribute specifying a `maven version range
+<https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN8903>`_ like ``[1.3,)`` for ``Version 1.5 or
```

Comment:
It might make more sense to link to the Javadocs of `Dependency.version` here https://jd.spongepowered.org/5.0.0/org/spongepowered/api/plugin/Dependency.html#version--